1.9/administration/upgrading.md

```diff
@@ -68,7 +68,7 @@ Proceed with upgrading every master node using the following procedure. When you
     $ curl -O http://<bootstrap_url>:<your_port>/dcos_install.sh
     ```
 
-1.  Uninstall DC/OS using the DC/OS Component Package Manager (Pkgpanda):
+1.  Uninstall pkgpanda:
```

Comment:
This does not uninstall pkgpanda, but uses pkgpanda to uninstall all the pkgpanda packages (aka "DC/OS system components", aka "dcos-*" systemd units)